app/src/ui/repositories-list/group-repositories.ts

```diff
@@ -84,9 +120,21 @@ export function groupRepositories(
   }
 
   // NB: This ordering reflects the order in the repositories sidebar.
-  addGroup('github')
-  addGroup('enterprise')
-  addGroup('other')
+  if (enableGroupRepositoriesByOwner()) {
+    const owners = gitHubOwners.reduce((acc, val) => {
```

Comment:
If we use a `Set<string>` above we don't need to do this reducing to eliminate duplicate keys:

```
const owners = [...gitHubOwners.keys()]
```

Reply:
good call!

Reply:
done in 0850f18